compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -269,6 +269,26 @@ def IREEGPU_MmaScheduleAttr : AttrDef<IREEGPU_Dialect, "MMASchedule"> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// Workgroup Reordering Attr
+
+def IREEGPU_WorkGroupReorderAttr: AttrDef<IREEGPU_Dialect, "WorkgroupReorderOptions">{
+  let mnemonic = "reorder_workgroups";
```

Comment:
nit: Can you make the naming between the IR mnemonic and the C++ class consistent? Maybe make both called `workgroup_reordering` and `WorkgroupReorderingAttr` respectively? It makes it easier to guess what the c++ class for a given attribute is later on.

Reply:
Sure, will do